data/json/mapgen/deserter_city/deserter_city_office.json

```diff
@@ -0,0 +1,317 @@
+    "method": "json",
+    "om_terrain": "deserter_city_office_2f",
+    "object": {
+      "fill_ter": "t_linoleum_gray",
+      "rows": [
+        "////////////////////////",
+        "/5gggg#####ww###wwww###`",
+        "/g````#...G..V.......f#`",
+        "/g````#hd.V..V..htth.fw`",
+        "/g````waf^V..V..htth..w`",
+        "/g````#####..G..htth..w`",
+        "/g````#...G..Va......^#`",
+        "/g````#hd.V..##########`",
+        "/g````waf.V..#.......a#`",
+        "/##########..#.ccccccc#`",
+        "/#fff#^......+........#`",
+        "/#...+.......#.....???#`",
+        "/########+#####VGGVVVV#`",
+        "/#T#T#T#yi#EEE#.......#`",
+        "/#+#+#+####EEE-......?w`",
+        "/#iiiii+..#EEH#......?#`",
+        "/#CSCSC#..#####......nw`",
+        "/#######.............?#`",
+        "/#CSCSC#..#####......?w`",
+        "/#iiiii+..#ZZZ#.......#`",
+        "/#+#+#+#..#ZZZ-..,##:##`",
+        "/#T#T#T#..#ZZZ#...#,.>#`",
+        "4#######ww#####ww,,#,##`",
+        "////////////////////////"
+      ],
+      "palettes": [ "deserter_city" ],
+      "terrain": { "g": "t_gutter", "`": "t_flat_roof", ":": "t_door_b" },
+      "place_npcs": [
+        { "class": "deserter", "x": 13, "y": 6 },
+        { "class": "deserter", "x": 9, "y": 19 },
+        { "class": "deserter", "x": 17, "y": 5 }
+      ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": "deserter_city_office_3f",
+    "object": {
+      "fill_ter": "t_linoleum_gray",
+      "rows": [
+        "////////////////////////",
+        "//////#################/",
+        "//////#iii*.......*iii#/",
+        "//////#iii%...c...%iii#/",
+        "//////#iii*...c...*iii#/",
+        "//////#****...c...****#/",
+        "//////#iii*...Q...*iii#/",
+        "//////#iii%...c...%iii#/",
+        "//////#iii*...c...*iii#/",
+        "/RRRRR#****...c...****#/",
+        "/Rssss#...............#/",
+        "/Rssss-...............#/",
+        "/*****############:####/",
+        "/*''''h''a#ZZZ#''x''''#/",
+        "/*'''ddo'f#ZZZ-'''''''w/",
+        "/*'''''''f#ZZZ#'''''''#/",
+        "/*''''''''#####'''''''w/",
+        "/*''''''''G'''G'''''''#/",
+        "/###''''''#####'''''''w/",
+        "/#S#''''''#ZZZ#'''''''#/",
+        "/#i+''''''#ZZZ-''Z##+##/",
+        "/#T#''''''#ZZZ#'',,Z''#/",
+        "4#################Z#Z##/",
+        "////////////////////////"
+      ],
+      "palettes": [ "deserter_city" ],
+      "terrain": {
+        "'": "t_floor",
+        "h": "t_floor",
+        "d": "t_floor",
+        "f": "t_floor",
+        "a": "t_floor",
+        "o": "t_floor",
+        "x": "t_floor",
+        ":": "t_door_metal_elocked",
+        "*": "t_ballistic_glass",
+        "%": [ "t_door_metal_o", [ "t_door_metal_interior_locked", 2 ] ]
+      },
+      "furniture": { "o": "f_desk" },
+      "items": {
+        "c": { "item": "science", "chance": 20, "repeat": [ 1, 2 ] },
+        "Q": { "item": "guns_energy", "chance": 100 },
```

Comment:
Energy weapons don't make much sense in this facility, middle of a town with just a small room and no infrastructure. Even if there were energy weapons, why aren't the soldiers using them then?